src/session-list-view.ts

```
@@ -0,0 +1,235 @@
+    // Lifted into the zone — no second copy in the timeline (user verdict:
+    // the duplicate was more noise than signal).
+    if (groupPinned && isPinned) continue;
+    const minor =
+      !isSearching &&
+      // An ungrouped pin must never fold into the minor group: pinning is an
+      // explicit "keep this", and a pinned session can still be a short
+      // untitled one that the junk predicate would happily fold away.
+      !isPinned &&
+      (hiddenSet.has(s.sessionId) ||
+        (activeDetectionReady &&
+          isMinorSession(s, hasCustomTitle(s.sessionId), hasPrLink(s.sessionId))));
```

ESLint

[error] Replace `s,·hasCustomTitle(s.sessionId),·hasPrLink(s.sessionId)` with `⏎············s,⏎············hasCustomTitle(s.sessionId),⏎············hasPrLink(s.sessionId),⏎··········`

(prettier/prettier)